config/routes.rb

```diff
@@ -12,4 +12,6 @@
   end
 
   resources :styles, only: [:index, :show]
+
+  resources :breweries, only: [:index, :show]
```

Comment:
What if a user wants to enter in a beer and you don't have that brewery in the database?